src/webkit.cs

```diff
@@ -708,23 +708,59 @@ partial interface DomDocument {
 		[Export ("cookie", ArgumentSemantic.Copy)]
 		string Cookie { get; set;  }
 
-		[Export ("body", ArgumentSemantic.Retain)]
+#if !XAMCORE_5_0
+		[Obsolete ("Use the 'Body' property instead.")]
+		[Wrap ("Body", IsVirtual = true)]
 		DomHtmlElement body { get; set;  }
+#endif
 
-		[Export ("images", ArgumentSemantic.Retain)]
+		[Export ("body", ArgumentSemantic.Retain)]
+		DomHtmlElement Body { get; set;  }
+
+#if !XAMCORE_5_0
+		[Obsolete ("Use the 'Images' property instead.")]
+		[Wrap ("Images", IsVirtual = true)]
 		DomHtmlCollection images { get;  }
+#endif
 
-		[Export ("applets", ArgumentSemantic.Retain)]
+		[Export ("images", ArgumentSemantic.Retain)]
+		DomHtmlCollection Images { get;  }
+
+#if !XAMCORE_5_0
+		[Obsolete ("Use the 'Applets' property instead.")]
+		[Wrap ("Applets", IsVirtual = true)]
 		DomHtmlCollection applets { get;  }
+#endif
 
-		[Export ("links", ArgumentSemantic.Retain)]
+		[Export ("applets", ArgumentSemantic.Retain)]
+		DomHtmlCollection Applets { get;  }
+
+#if !XAMCORE_5_0
+		[Obsolete ("Use the 'Links' property instead.")]
+		[Wrap ("Links", IsVirtual = true)]
 		DomHtmlCollection links { get;  }
+#endif
 
-		[Export ("forms", ArgumentSemantic.Retain)]
+		[Export ("links", ArgumentSemantic.Retain)]
+		DomHtmlCollection Links { get;  }
+
+#if !XAMCORE_5_0
+		[Obsolete ("Use the 'Forms' property instead.")]
+		[Wrap ("Forms", IsVirtual = true)]
 		DomHtmlCollection forms { get;  }
+#endif
 
-		[Export ("anchors", ArgumentSemantic.Retain)]
+		[Export ("forms", ArgumentSemantic.Retain)]
+		DomHtmlCollection Forms { get;  }
+
+#if !XAMCORE_5_0
+		[Obsolete ("Use the 'Anchors' property instead.")]
+		[Wrap ("Anchors", IsVirtual = true)]
 		DomHtmlCollection anchors { get;  }
+#endif
+
+		[Export ("anchors", ArgumentSemantic.Retain)]
+		DomHtmlCollection Anchors { get;  }
```

Comment:
This is not a big issue at all (no need to change this PR), but I like diffs that show no removed lines (they're much easier to review). This isn't always possible of course, but once in a while it's possible to reorganize the new code so that the resulting diff is easier to read, and I believe this to be one such case.

This is the old code:

```cs
		[Export ("anchors", ArgumentSemantic.Retain)]
		DomHtmlCollection anchors { get;  }
```

and if you did the change like this:

```cs
		[Export ("anchors", ArgumentSemantic.Retain)]
		DomHtmlCollection Anchors { get;  }

#if !XAMCORE_5_0
		[Obsolete ("Use the 'Anchors' property instead.")]
		[Wrap ("Anchors", IsVirtual = true)]
		DomHtmlCollection anchors { get;  }
#endif
```

I believe the diff would only show added lines, hopefully (I haven't tested this, so I might be wrong!) something like this:

```diff
 		[Export ("anchors", ArgumentSemantic.Retain)]
+		DomHtmlCollection Anchors { get;  }
+ 
+ #if !XAMCORE_5_0
+		[Obsolete ("Use the 'Anchors' property instead.")]
+		[Wrap ("Anchors", IsVirtual = true)]
		DomHtmlCollection anchors { get;  }
+#endif
```

In fact the complication of unnecessary removed lines in the diff is showcased very well in this PR, because if you just look at a subset of the diff:

```diff
-		[Export ("images", ArgumentSemantic.Retain)]
+		[Export ("body", ArgumentSemantic.Retain)]
```

You'll see that something doesn't look right ("Why is the selector changing? That's not what this PR is supposed to do!"). You have to study the diff much harder to realize that effectively no selectors are changing.

Reply:
Thank you for suggesting this! Even when I was reviewing the diff it was confusing to have the unnecessary changes in the way. I'll keep this in mind for future changes.